tests/functional/pyocf/utils.py

```diff
@@ -88,6 +88,10 @@ def __int__(self):
     def __index__(self):
         return self.bytes
 
+    @classmethod
+    def from_sectors(cls, value):
```

Comment:
Take a look at line :116 of this file ;)

Reply:
Whoopsie, removed